lib/decision_review/service.rb

```diff
@@ -60,23 +46,44 @@ def get_higher_level_reviews_intake_status(uuid)
     def get_higher_level_reviews(uuid)
       with_monitoring_and_error_handling do
         raw_response = perform(:get, "higher_level_reviews/#{uuid}", nil)
-        DecisionReview::Responses::Response.new(raw_response.status, raw_response.body, 'review')
+        DecisionReview::Responses::Response.new(raw_response.status, raw_response.body, 'HLR-SHOW-RESPONSE-200')
```

Comment:
was there a reason you chose all caps for this string?  Seems unusual to me.

Reply:
the way vets_json_schema is set up is that it automatically upcases schema names.
yeah, i don't like it either